includes/Handlers/Product.php

```diff
@@ -1657,13 +1657,14 @@ private static function update_price_money( $product, \Square\Models\CatalogObje
 	 */
 	public static function get_parent_product_id_by_variation_id( $variation_id ) {
 		global $wpdb;
-
+		$tablename = $wpdb->prefix.'posts';
+		
 		return $wpdb->get_var(
 			$wpdb->prepare(
 				"
 				SELECT pr.post_parent
-				FROM wp_posts pr
-				INNER JOIN wp_posts pp ON pp.ID = pr.post_parent
+				FROM $tablename pr
+				INNER JOIN $tablename pp ON pp.ID = pr.post_parent
```

Comment:
I'd suggest instead of saving the table name into a new variable, we just add the prefix inline to match how we do this in other places:

```suggestion
				FROM {$wpdb->prefix}posts pr
				INNER JOIN {$wpdb->prefix}posts pp ON pp.ID = pr.post_parent
```